.github/workflows/frequency_specs.yml

```diff
@@ -39,36 +39,38 @@ jobs:
         uses: Swatinem/rust-cache@v1
         with:
```

Comment:
Updates Here
- Remove need for generating rococo-local binary 
- Create rococo and mainnet binaries separately